src/main/java/org/jfrog/hudson/pipeline/docker/utils/DockerUtils.java

```diff
@@ -333,4 +337,22 @@ private static void closeQuietly(DockerClient dockerClient) {
             }
         }
     }
+
```

Comment:
Let's rename this method to "validateResponse".
Let's also change the exception message to: "Failed while trying to set properties <THE PROPERTIES> on docker layer: <LAYER FULL PATH>". Received HTTP response code <THE CODE> while expecting 204."
This method can also be moved into the DockerImage class and be private.